lms/djangoapps/student_account/views.py

```diff
@@ -133,9 +133,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'third_party_auth_hint': third_party_auth_hint or '',
             'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
             'support_link': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK),
-            'password_reset_support_link': configuration_helpers.get_value(
-                'PASSWORD_RESET_SUPPORT_LINK', settings.SUPPORT_SITE_LINK  # Intentional default.
-            ),
```

Comment:
Also, we'll need to add `PASSWORD_RESET_SUPPORT_LINK` to [edx/configuration](https://github.com/edx/configuration) and [edx/edx-internal](https://github.com/edx/edx-internal).

See these similar PRs for adding `ACCOUNT_ACTIVATION_SUPPORT_LINK` to the repos:
https://github.com/edx/configuration/pull/3886
https://github.com/edx/edx-internal/pull/81